t/rose-macro/test_header

```diff
@@ -77,3 +77,15 @@ function teardown() {
     rm -rf $TEST_DIR/run
     rm -rf $TEST_DIR/config/meta
 }
+
+function grep_count_ok() {
```

Comment:
This is useful.

It makes Matt happy to have double quotes around environment variables like `$FILE` and `$TEST_KEY` in case there are spaces inside.